bin/vm/tests.js

```diff
@@ -254,23 +249,15 @@ ava.serial('vm serialport log', async t => {
     await common.cleanup();
 });
 
-const sshVmPassword = (ip, password, command) => ssh.execute(command, {
```

Comment:
I extracted this code to the library because I intended to use it again. By the way, I simplified it by using DNS.